newjersey/navigator.business.nj.gov
# Move shared factories to shared file #183973645

## Description

We wanted to consolidate a bunch of our factories that are repeated between the web and api factories into our shared factories.



### Ticket



[Replace with ticket_id](https://www.pivotaltracker.com/story/show/183973645)

### Approach

Basically a lot of copying and pasting and re building and re running of tests. 



### Steps to Test

Make sure all tests, run. Glance at code and make sure the new position of code makes sense



### Notes

It seemed like the only way to get imports to work in the shared factories was via relative imports. 

Probably could and should have been broken up into more commits but it was a little late to fix when I realized this

Also because of how this is structured a lot of these are available via the @shared tag alone rather than the @shared/tests tag. So they can be referenced via slightly different imports. I thought this wasn't something that needed to be 100% consistent in every place. 



## Code author checklist
